<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>LoginWindow</class>
    <widget class="QMainWindow" name="LoginWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>300</width>
                <height>350</height>
            </rect>
        </property>
        <property name="maximumSize">
            <size>
                <width>300</width>
                <height>350</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Авторизация</string>
        </property>
        <property name="windowIcon">
            <iconset>
                <normaloff>img/logo.ico</normaloff>img/logo.ico
            </iconset>
        </property>
        <property name="tabShape">
            <enum>QTabWidget::Rounded</enum>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QWidget" name="verticalLayoutWidget">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>0</y>
                        <width>261</width>
                        <height>101</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                    <item>
                        <widget class="QLabel" name="aircraft_logo">
                            <property name="text">
                                <string/>
                            </property>
                            <property name="pixmap">
                                <pixmap>img/Delta-A350.png</pixmap>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget_2">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>110</y>
                        <width>62</width>
                        <height>46</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                        <widget class="QLabel" name="Login">
                            <property name="text">
                                <string>Login:</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget_3">
                <property name="geometry">
                    <rect>
                        <x>80</x>
                        <y>110</y>
                        <width>201</width>
                        <height>41</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                        <widget class="QLineEdit" name="login_input">
                            <property name="maxLength">
                                <number>30</number>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget_4">
                <property name="geometry">
                    <rect>
                        <x>80</x>
                        <y>155</y>
                        <width>201</width>
                        <height>51</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                        <widget class="QLineEdit" name="password_input">
                            <property name="maxLength">
                                <number>30</number>
                            </property>
                            <property name="echoMode">
                                <enum>QLineEdit::Password</enum>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget_5">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>160</y>
                        <width>62</width>
                        <height>46</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_8">
                    <item>
                        <widget class="QLabel" name="Pass">
                            <property name="text">
                                <string>Pass:</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget_6">
                <property name="geometry">
                    <rect>
                        <x>20</x>
                        <y>210</y>
                        <width>261</width>
                        <height>41</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                    <item>
                        <widget class="QLabel" name="error_box">
                            <property name="text">
                                <string></string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget_7">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>260</y>
                        <width>101</width>
                        <height>31</height>
                    </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_10">
                    <item>
                        <widget class="QPushButton" name="login_submit_btn">
                            <property name="text">
                                <string>Login</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>300</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <action name="actionMain">
            <property name="text">
                <string>Main</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
